packages/provider/src/HocuspocusProvider.ts

```diff
@@ -447,10 +443,6 @@ export class HocuspocusProvider extends EventEmitter {
 
     const documentName = message.readVarString()
 
-    if (documentName !== this.configuration.name) {
-      return // message is meant for another provider
-    }
-
     message.writeVarString(documentName)
```

Comment:
As an aside, I'm not sure why it is required to read the documentName now that this function does not use it. However, removing the read and write breaks the message. I even tried messing around with the encoder and decoder positions but could not get it to work without leaving the read + write as-is. There is clearly a side effect here that I do not understand.

Reply:
yea, that won't work because of the position. You'd need to manually read the first byte, and then move the pointer by the value of it (first byte contains the length of the string). I guess we can keep it like this though, the performance impact should be neglectable.